### ISSUE TITLE
Update the debian repo?

### ISSUE THREAD

**Issue by lorentrogers** [QTodoTxt/QTodoTxt#315]
Looks like 1.4 is the latest in the deb repo: http://dl.bintray.com/mnantern/deb/

Could we update this with the latest?

**Comment by oroulet** [comment]
mnantern maintained that one. If he is not doing anything, we should just
remove the link

**Comment by lorentrogers** [comment]
Thinking about it, it probably makes more sense to have a shared account between the maintainers. I'll remove it for now, and if there's interest we could set up a new "QTodoTxt" account.

**Comment by PKEv** [comment]
@mNantern can help?

**Comment by oroulet** [comment]
I am not sure we need a repository anyway. As long as we dot not have any
complicated dependency

**Comment by PKEv** [comment]
what about web service like: http://docs.drone.io/newproject.html ?

**Comment by lorentrogers** [comment]
I agree that some form of CI would be useful. If we could have it maintain the debian repo on its own, that'd be great. (Another useful one is Hound: https://houndci.com/)

For now, would it make sense to just link to the github page, as I've done in #316 ?

**Comment by PKEv** [comment]
@lorentrogers how i can see https://houndci.com can't deploy only comment

**Comment by lorentrogers** [comment]
@PKEv it's just a code style checker. It doesn't deploy anything, but it's useful for maintaining a clean development repo. I figured I'd mention it, since we're talking about CI servers.

**Comment by lorentrogers** [comment]
Hmm. Now that I look at the site again, it seems they don't support Python... Never mind!
